resim/metrics/python/metrics_test.py

```diff
@@ -1658,6 +1658,137 @@ def test_external_file_metrics_data_pack(self) -> None:
         self.assertEqual(msg.name, metrics_data.name)
         self.assertEqual(msg.external_file.path, filename)
 
+    def test_event_eq(self) -> None:
+        # SETUP
+        metric_1 = metrics.ScalarMetric(
+            name="test_metric_1",
+            value=24.0)
+        metric_2 = metrics.ScalarMetric(
+            name="test_metric_2",
+            value=24.0)
```

Comment:
[nit] Might make sense to give these different values in case they're somehow being mixed up by the code, which would be highly unlikely.

Reply:
yeah, makes sense!